pkg/grafana/httpclient.go

```diff
@@ -0,0 +1,10 @@
+package grafana
+
+import (
+	"net/http"
+	"time"
+)
+
+func NewHttpClient() *http.Client {
+	return &http.Client{Timeout: 10 * time.Second}
```

Comment:
Would be even nicer if this were configurable

Reply:
I second this, might render grizzly unusable in bad network conditions otherwise

Reply:
`GRIZZLY_HTTP_TIMEOUT` with a default of 10s?

Reply:
Seems reasonable to me

Reply:
Done